rudder-core/src/main/scala/com/normation/cfclerk/services/impl/Cf3PromisesFileWriterServiceImpl.scala

```diff
@@ -347,26 +357,21 @@ class Cf3PromisesFileWriterServiceImpl(
       if(technique.providesExpectedReports) {
         techniqueBundles match {
           case Seq() => Seq()
-          case head +: tail => Bundle(s"${NCF_REPORT_DEFINITION_BUNDLE_NAME}(${head.name})") +: head +: tail
+          case (p,b) +: tail => (p, Bundle(s"${NCF_REPORT_DEFINITION_BUNDLE_NAME}(${b.name})")) +: (p,b) +: tail
         }
       } else {
         techniqueBundles
       }
     }
 
+    List(
+      SystemVariable(systemVariableSpecService.get("INPUTLIST"), Seq(inputs.distinct.mkString("\"", "\",\"", "\"")))
+    , SystemVariable(systemVariableSpecService.get("BUNDLELIST"), Seq(bundleSeq.map( _._2.name).mkString(", ", ", ", "")))
+    , SystemVariable(systemVariableSpecService.get("RUDDER_DIRECTIVES_SEQUENCE"),
+        Seq(bundleSeq.map { case (promisee, bundle) => s""" "${promisee} usebundle => ${bundle.name};" """}.mkString( "\n"))
```

Comment:
This should be promiser, not promisee. Promisee is a whole different concept in CFEngine. See http://syslog.me/2013/05/21/the-value-of-the-promisee-in-cfengine-promises/.